data/avgi/avgi ships.txt

```diff
@@ -0,0 +1,2142 @@
+		left
+	"steering engine" 0 51
+		angle -30
+		right
+	"steering engine" 0 51
+		angle 30
+		left
+
+	leak "flame" 30 90
+	explode "tiny explosion" 12
+	explode "small explosion" 9
+	"final explode" "final explosion small"
+
+	description "The Tremoros-class Interceptor is a small autonomous ship designed to support patrol and interdiction missions. With an emphasis on speed, it can rocket past most adversaries even as it brings its guns to bear, making it exceedingly difficult to hit."
+	description "The Tremoros has also seen use as an escort for smaller merchant convoys, with the ability to intercept any enemies that might pose a threat before they can come within range of vulnerable civilian ships."
+
+
+# Light Warships
+
+ship "Prismaios"
+	plural "Prismaios"
+	sprite "ship/avgi prismaios"
+	thumbnail "thumbnail/avgi prismaios"
+	swizzle 0
+	attributes
+		category "Light Warship"
+		licenses
+			"Avgi Military"
+		"cost" 240000
+		"shields" 1440
+		"hull" 7200
+		"required crew" 2
+		"bunks" 6
+		"mass" 120
+		"drag" 1.2
+		"heat dissipation" 1.8
+		"fuel capacity" 1200
+		"cargo space" 36
+		"outfit space" 330
+		"weapon capacity" 96
+		"engine capacity" 96
+		"anchor point" 2
+		weapon
+			"blast radius" 240
+			"shield damage" 1500
+			"hull damage" 900
+			"hit force" 1200
+	outfits
+		"Phased Array Laser (Green)"
+		"Speckle Turret" 2
+		"Speck Round" 14400
+		"Sidereal Launcher" 2
+		"Sidereal Missile" 84
+		"Electromotive Rifle" 6
+
+		"Luminous Fusion Core"
+		"Ultracapacitor Bank"
+		"Diffuse Deflector"
+		"Optical Lasing Generator"
+		"Multispectral Scanner"
+		"Electrostatic Ramscoop"
+		
+		"Z-360 Fusion Torch"
+		"XY-420 Radial Thrusters"
+		"Hyperdrive"
+
+	engine 0 108
+	"reverse engine" -6 99
+		angle -45
+	"reverse engine" 6 99
+		angle 45
+	"steering engine" -6 -66
+		angle 90
+		right
+	"steering engine" 6 -66
+		angle -90
+		left
+	"steering engine" -3 99
+		angle 90
+		left
+	"steering engine" 3 99
+		angle -90
+		right
+	gun -9 -48
+	gun 9 -48
+	gun -9 33 "Sidereal Launcher"
+		angle -90
+	gun 9 33 "Sidereal Launcher"
+		angle 90
+	turret -9 18 "Speckle Turret"
+	turret 9 18 "Speckle Turret"
+	turret 0 90 "Phased Array Laser (Green)"
+
+	leak "leak" 30 60
+	leak "flame" 30 90
+	explode "tiny explosion" 12
+	explode "small explosion" 15
+	explode "medium explosion" 15
+	"final explode" "final explosion medium"
+
+	description "The Prismaios-class Frigate is a recent, streamlined design, praised by many Twilight Guard captains as a highly successful addition to Avgi strike fleets. New construction techniques allow for unprecedented gains in structural integrity for such a lightweight frame, allowing the frigate to serve on the front lines against the Aberrant as a fast, versatile, multipurpose warship."
+
+
+ship "Interferos"
+	plural "Interferos"
+	sprite "ship/avgi interferos"
+	thumbnail "thumbnail/avgi interferos"
+	swizzle 0
+	attributes
+		category "Light Warship"
+		licenses
+			"Avgi Civilian"
+		"cost" 240000
```

Comment:
```suggestion
		"cost" 1380000
```
Similar to human light warships, but skewing toward the low end somewhat given the design's focus on inexpensive construction.